hs_core/views/__init__.py

```diff
@@ -617,6 +653,8 @@ def delete_resource(request, shortkey, *args, **kwargs):
                               resource_shortkey=shortkey, resource=res,
                               resource_title=res_title, resource_type=res_type, **kwargs)
 
+    web_services.update(shortkey)
```

Comment:
Need to put web_services.update() call into a celery task so it runs in a background thread so as not to block the main thread, especially given the calling functions are called very frequently.